prowler/lib/outputs/finding.py

```diff
@@ -0,0 +1,191 @@
+                output_data["account_name"] = check_output.subscription
+                output_data["resource_name"] = check_output.resource_name
+                output_data["resource_uid"] = check_output.resource_id
+                output_data["region"] = check_output.location
+
+            elif provider.type == "gcp":
+                output_data["auth_method"] = f"Principal: {output_data['auth_method']}"
+                output_data["account_uid"] = provider.projects[
+                    check_output.project_id
+                ].id
+                output_data["account_name"] = provider.projects[
+                    check_output.project_id
+                ].name
+                output_data["account_tags"] = provider.projects[
+                    check_output.project_id
+                ].labels
+                output_data["resource_name"] = check_output.resource_name
+                output_data["resource_uid"] = check_output.resource_id
+                output_data["region"] = check_output.location
+
+                if (
+                    provider.projects
+                    and check_output.project_id in provider.projects
+                    and getattr(
+                        provider.projects[check_output.project_id], "organization"
+                    )
+                ):
+                    output_data["account_organization_uid"] = provider.projects[
+                        check_output.project_id
+                    ].organization.id
+                    # TODO: for now is None since we don't retrieve that data
+                    output_data["account_organization"] = provider.projects[
+                        check_output.project_id
+                    ].organization.display_name
+
+            elif provider.type == "kubernetes":
+                if provider.identity.context == "In-Cluster":
+                    output_data["auth_method"] = "in-cluster"
+                else:
+                    output_data["auth_method"] = "kubeconfig"
+                output_data["resource_name"] = check_output.resource_name
+                output_data["resource_uid"] = check_output.resource_id
+                output_data["account_name"] = f"context: {provider.identity.context}"
+                output_data["region"] = f"namespace: {check_output.namespace}"
+
+            # check_output Unique ID
+            # TODO: move this to a function
+            # TODO: in Azure, GCP and K8s there are fidings without resource_name
+            output_data["finding_uid"] = (
+                f"prowler-{provider.type}-{check_output.check_metadata.CheckID}-{output_data['account_uid']}-{output_data['region']}-{output_data['resource_name']}"
+            )
+
+            finding_output = cls(**output_data)
+        except Exception as error:
+            logger.error(
+                f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+        else:
+            return finding_output
```

Comment:
The `else` block doesn't add anything here. It is more readable this way.

```suggestion
            return cls(**output_data)
        except Exception as error:
            logger.error(
                f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
            )
```

Reply:
Done